tests/unit/test_time_step_control.py

```diff
@@ -1,3 +1,4 @@
+import numpy as np
```

Comment:
Could we have a summary of the tests here at the top of the file, please?

Reply:
Sure. I have included a brief summary of the tests as requested.